mailpoet/assets/js/src/automation/integrations/woocommerce-subscriptions/steps/subscription-trial-ended/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { __ } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { Icon } from './icon';
+import { PremiumModalForStepEdit } from '../../../../../common/premium-modal';
+
+const keywords = [
+  __('woocommerce', 'mailpoet'),
+  __('subscription', 'mailpoet'),
+  __('trial', 'mailpoet'),
+  __('ended', 'mailpoet'),
+];
+export const step: StepType = {
+  key: 'woocommerce-subscriptions:trial-ended',
+  group: 'triggers',
+  title: () => __('Woo Subscription trial ended', 'mailpoet'),
+  description: () =>
+    __('Start the automation when a subscription trial ended.', 'mailpoet'),
```

Comment:
`when a subscription trial ends` sounds better to me

Reply:
https://github.com/mailpoet/mailpoet/pull/5264/commits/d24111eda686d0e5643dd27b42db2c359aac3849